wireless_modules/mqtt_test/boot.py

```
@@ -0,0 +1,34 @@
+import config 
```

Any chance we can catch an error if `config.py` does not exist and/or cannot be found? Only asking because it will happen and people will ask you how to fix it!

I would re-raise the exception with an additional message stating to create a `config.py` script.

ALSO, can you update `README.md` with setup and how to run this?

---

I was waiting for you to write this @nix-chu

---

Just a thought, should this be a json or .env config file rather than an actual python file? Not sure if that'll work in micropython though

---

> I was waiting for you to write this @nix-chu

@khanguslee config file or a readme?

---

I believe @Blake-Haydon said `.env` files don't work in micropython. IDM a JSON file as well?

@nix-chu Oh I meant the suggestion since you said the same thing in my PR. 😄

---

I think actual python is good, as it is handy to initiate sensors and LEDs in this file.

---

> Any chance we can catch an error if config.py does not exist and/or cannot be found? Only asking because it will happen and people will ask you how to fix it!

I have added @angustrau build script so hopefully that doesn't happen

---

> I have added @angustrau build script so hopefully that doesn't happen

Not quite sure what you're referring to there. Just want to point out how this could be done quite easily if you're unsure about it:
```python
try:
    import config
except ImportError:
    print("error message here")
```

---

Oops my bad, I read it wrong, will add in the try catch now.